ext/spl/spl_fixedarray.c

```diff
@@ -358,6 +358,20 @@ static zval *spl_fixedarray_object_read_dimension(zval *object, zval *offset, in
 
 	intern = Z_SPLFIXEDARRAY_P(object);
 
+	if (type == BP_VAR_IS) {
```

Comment:
Might it not be necessary in the (admittedly rather odd) case where offsetGet has been overridden but offsetHas not? In that case we should still perform the isset based on default semantics (or not?)